contracts/swapHandlers/ISwapHandler.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity ^0.8.0;
+
+interface ISwapHandler {
+    /// @notice Params for swaps using SwapHub contract and swap handlers
+    /// @param underlyingIn sold token address
+    /// @param underlyingOut bought token address
+    /// @param mode type of the swap: 0 for exact input, 1 for exact output
+    /// @param amountIn amount of token to sell. Exact value for exact input, maximum for exact output
+    /// @param amountOut amount of token to buy. Exact value for exact output, minimum for exact input
+    /// @param exactOutTolerance Maximum difference between requested amountOut and received tokens in exact output swap
```

Comment:
might be explicitly stated that `exactOutTolerance` is ignored for exact input swap

Reply:
:+1: